bindings/python/tests/ExtWrenchesAndJointTorquesEstimatorUnitTest.py

```diff
@@ -0,0 +1,122 @@
+import sys
+
+sys.path.append("../../python/")
+sys.path.append("../../../lib/python/")
+sys.path.append("../../../lib/python/Debug/")
+
+import os
+import unittest
+import idyntree.swig as iDynTree;
+import numpy as np
+import random
+
+class ExtWrenchesAndJointTorquesEstimatorUnitTest(unittest.TestCase):
+    def tol(self):
+        return 1e-10;
+
+    def places(self):
+        return 7;
+
+    def checkApproxEqual(self,val1,val2,msg):
+        self.assertAlmostEqual(val1,val2,self.places(),msg);
+
+
+    def checkVecApproxEqual(self,p1,p2,msg):
+        msgMore = "val1 = " + str(p1) + " and val2 = " + str(p2)
+        for i in range(0,len(p1)):
+            self.checkApproxEqual(p1[i],p2[i],msg+":"+msgMore);
+
+    def testcomputeSubModelMatrixRelatingFTSensorsMeasuresAndKinematics(self):
+
+        # Define considered joints
+        consideredJoints = iDynTree.StringVector()
+        consideredJoints.push_back("torso_pitch")
+        consideredJoints.push_back("torso_roll")
+        consideredJoints.push_back("torso_yaw")
+        consideredJoints.push_back("neck_pitch")
+        consideredJoints.push_back("neck_roll")
+        consideredJoints.push_back("neck_yaw")
+        consideredJoints.push_back("l_shoulder_pitch")
+        consideredJoints.push_back("l_shoulder_roll")
+        consideredJoints.push_back("l_shoulder_yaw")
+        consideredJoints.push_back("l_elbow")
+        consideredJoints.push_back("r_shoulder_pitch")
+        consideredJoints.push_back("r_shoulder_roll")
+        consideredJoints.push_back("r_shoulder_yaw")
+        consideredJoints.push_back("r_elbow")
+        consideredJoints.push_back("l_hip_pitch")
+        consideredJoints.push_back("l_hip_roll")
+        consideredJoints.push_back("l_hip_yaw")
+        consideredJoints.push_back("l_knee")
+        consideredJoints.push_back("l_ankle_pitch")
+        consideredJoints.push_back("l_ankle_roll")
+        consideredJoints.push_back("r_hip_pitch")
+        consideredJoints.push_back("r_hip_roll")
+        consideredJoints.push_back("r_hip_yaw")
+        consideredJoints.push_back("r_knee")
+        consideredJoints.push_back("r_ankle_pitch")
+        consideredJoints.push_back("r_ankle_roll")
+
+        # Load model
+        estimator = iDynTree.ExtWrenchesAndJointTorquesEstimator()
+        testModel = os.path.join(os.path.dirname(__file__), "../../../../src/tests/data/iCubDarmstadt01.urdf")

```

Comment:
Is the test supposed to run from its current directory?

In this case the model file should be:

```
testModel = os.path.join(os.path.dirname(__file__), "../../../src/tests/data/iCubDarmstadt01.urdf")
```

Reply:
Sorry, I thought I replied. No, that file is copied in `build/bindings/python/tests`  to run. Indeed, that is confusing, I opened https://github.com/robotology/idyntree/issues/1113 to solve this.